lms/envs/common.py

```diff
@@ -2219,14 +2221,6 @@
 ENABLE_CREDIT_ELIGIBILITY = True
 FEATURES['ENABLE_CREDIT_ELIGIBILITY'] = ENABLE_CREDIT_ELIGIBILITY
 
-################ Enable JWT auth ####################
-# When this feature flag is set to False, API endpoints using
-# JSONWebTokenAuthentication will reject requests using JWT to authenticate,
-# even if those tokens are valid. Set this to True only if you need those
-# endpoints, and have configured settings 'JWT_AUTH' to override its default
-# values with secure values.
-FEATURES['ENABLE_JWT_AUTH'] = False
```

Comment:
I haven't had any additional conversations. The original issue seemed to lie with having an insecure value for `JWT_SECRET_KEY`. This is why I removed the default of `None`. The default from the package is the value of `SECRET_KEY`. This default resolves the issue for which the feature flag was implemented.